nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -342,6 +342,13 @@ bool CollisionMonitor::configureSources(
         r->configure();
 
         sources_.push_back(r);
+      } else if (source_type == "polygon_source") {
```

Comment:
Why not just `polygon`? (param + file name + class name)

Reply:
Just made it the same as the filename/class but you're right, doesn't have to be, changing